pkg/rhelv2/rpm/rpm_test.go

```diff
@@ -45,11 +45,11 @@ func Test_listFeatures(t *testing.T) {
 		isActiveVulnMgmt bool
 
 		// Expected values for assertion.
-		sampleExpectedPkgs []*database.RHELv2Package
-		unexpectedPkgs     []*database.RHELv2Package
-		expectedCPEs       []string
```

Comment:
do we test the mapping from content set to CPE in repo2cpe? If not, we should do that there, so we don't lose this test

Reply:
Do you mean https://github.com/stackrox/scanner/blob/0ee22232a00d3ca6d5ea53356d3a33bf9ca00428/pkg/repo2cpe/mapping_test.go#L11